Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cargo-llvm-cov"
-version = "0.6.4" #publish:version
+version = "0.6.5" #publish:version
```

Comment:
Please revert the version update. That will be handled automatically during release.